exercises/D-logical-operators/exercise2.js

```diff
@@ -5,8 +5,30 @@
   Update the code so that you get the expected result.
 */
 
-function isNegative() {}
+function isNegative(number) {
+  return (number < 0) ?  true : false;
```

Comment:
however it works fine, you don't really need the ternary operator here, number < 0 works well